res/i18n/fr/labels.json

```diff
@@ -521,7 +521,7 @@
 		"pan_right": "Pan droite",
 		"peak": "Pic",
 		"phase": "Phase",
-		"positive_width": "Largeur positive",
+		"positive_width": "Largeur positif",
```

Comment:
Same than line 515. "largeur" is a feminine noun, hence the adjective needs to match it.
Please revert.

Reply:
I've change this cause in my head : it's the width of the signal, and in french "signal" is a masculin word.

Reply:
This adjective is related to the noun "largeur", so it definitely need to match that, hence it's "largeur positive".
Ping @wargreen.